test/generators/brakeman_install_generator_test.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'generators/boring/brakeman/install/install_generator'
+
+class AuditInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Brakeman::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_add_brakeman_gem
```

Comment:
Let's update to "test_should_configure_brakeman_gem" and check both install and as well as configuration inside it, no need to add two separate methods for that. In every test, we need to add one method that looks at the happiest path for the generator i.e. configure_<gem_name>_gem